03-other-topics/005-k8s-basics.md

```diff
@@ -0,0 +1,81 @@
+---
+category: cloud-platform
+expires: 2019-01-01
+---
+# Kubernetes Basics - Links to helpful beginner resources
+
+The purpose of this page is to serve as a small catalogue of links to basic Kubernetes concepts and learning resources.
+
+## The Concept
+
+Kubernetes is very often referred to as **"K8s"**, purely for the reason, that typing Kubernetes over and over gets quite taxing.
+
+If Kubernetes is a totally new concept to you, then firstly you should probably check out a few of the resources linked below:
+
+Official Website: 
+https://kubernetes.io/
+
+Offical Docs: 
+https://kubernetes.io/docs/concepts/
+
+Quick concept video: 
+https://www.youtube.com/watch?v=IMOZCDhH7do
+
+## Usage Platforms
```

Comment:
Do users really need to know about this?
I'm not sure this section qualifies as being helpful for beginners.

Reply:
I guess maybe not, happy to take it out.